.azure-pipelines.yml

```diff
@@ -84,6 +85,16 @@ jobs:
         displayName: "PyTest (treat warnings as errors)"
         condition: eq(variables['TEST_TYPE'], 'strict-warning')
 
+      # TODO: remove: https://github.com/scverse/anndata/issues/1451
+      - task: PythonScript@0
+        inputs:
+          scriptSource: inline
+          script: |
+            import sys, xml.etree.ElementTree as ET
+            results = ET.parse("./test-data/test-results.xml").findall("./*/*")
+            sys.exit(0 if len(results) > 100 else f"Error: only {len(results)} tests run")
```

Comment:
If you wanted to bump this, we're at 3892 tests right now. So something around there. Should be fine otherwise.